### ISSUE TITLE
Deleting a local style is invalid !!! lv_obj_remove_local_style_prop()

### ISSUE THREAD

**Issue by Yangyup** [kisvegabor/lvgl#3581]
### Perform all steps below and tick them with [x]
- [ ] Read the [FAQ](https://docs.lvgl.io/master/intro/index.html#faq)
- [ ] Check the related part of the [Documentation](https://docs.lvgl.io/)
- [ ] Update lvgl to the latest version
- [ ] Reproduce the issue in a [Simulator](https://docs.lvgl.io/latest/en/html/get-started/pc-simulator.html)

### Describe the bug

LVGL V8.0.1 
Setting the style function: lv_obj_set_style_bg_color();
Delete style functions:lv_obj_remove_local_style_prop();

I don't understand these two functions. Setting succeeds but deleting fails. Could you give me some help to understand the cause of this mistake

### To Reproduce

Embedded system (Tina system)
```c
static void my_event_cb(lv_event_t * e)
{
    lv_obj_t * obj = lv_event_get_target(e);        
    lv_event_code_t code = lv_event_get_code(e);    
    lv_obj_t * label = lv_event_get_user_data(e);   
    bool res = false;

    switch(code){
        case LV_EVENT_PRESSED:
            lv_label_set_text(label, "LV_EVENT_PRESSED");
            lv_obj_set_style_bg_color(obj, lv_color_hex(0xc43e1c), 0);   //Setting the style function
            printf("LV_EVENT_PRESSED\n");
            break;
	case LV_EVENT_LONG_PRESSED:
            lv_label_set_text(label, "LV_EVENT_LONG_PRESSED");
            lv_obj_set_style_bg_color(obj, lv_color_hex(0x4cbe37), 0);   //Setting the style function
            printf("LV_EVENT_LONG_PRESSED\n");
            break;
        case LV_EVENT_CLICKED:
            lv_label_set_text(label, "LV_EVENT_CLICKED");
            res = lv_obj_remove_local_style_prop(obj, LV_STYLE_BG_COLOR, 0);  //Delete style functions
	    if(res)
            {
                printf("remove style succes\n");
            }
            else
            {
                printf("remove style fail\n");
            }
            printf("LV_EVENT_CLICKED\n");
            break;
        default:
            //printf("NONE\n");
            break;
    }
}

void lv_100ask_demo_course_2_2_6(void)
{
    lv_obj_t * obj = lv_obj_create(lv_scr_act());

    lv_obj_t * label = lv_label_create(lv_scr_act());
    lv_label_set_text(label, "test"); 
    lv_obj_center(label);               

    //The callback function
    lv_obj_add_event_cb(obj, my_event_cb, LV_EVENT_ALL, label);
}
```

### Expected behavior

Event1 :
LV_EVENT_PRESSED Setting the style function

Event2:
LV_EVENT_LONG_PRESSED  Setting the style function

Event3:
 LV_EVENT_CLICKED Delete style functions

### Screenshots or video

![0996c7dc8f549a52f5483da0dd13a3c](https://user-images.githubusercontent.com/106371849/184820457-9f4bf4cb-2662-4513-9090-be6fbd45639a.jpg)

https://user-images.githubusercontent.com/106371849/184820481-a60f8821-b64d-4ea2-befe-9809dc3f2065.mp4

**Comment by kisvegabor** [comment]
It was a bug in lvgl. I've fixed it in both `release/v8.3` and `mater` branches.

**Comment by Yangyup** [comment]
@kisvegabor Thank you very much. Now I've got it
